web-client/src/presenter/computeds/confirmInitiateServiceModalHelper.js

```diff
@@ -63,7 +55,7 @@ export const confirmInitiateServiceModalHelper = (get, applicationContext) => {
     formattedCaseDetail.isLeadCase &&
     showConsolidatedOptions &&
     consolidatedCasesPropagateDocketEntriesFlag &&
-    !eventCodesNotCompatibleWithConsolidation.includes(eventCode) &&
+    !NON_MULTI_DOCKETABLE_EVENT_CODES.includes(form.eventCode) &&
```

Comment:
I had changed this to check for eventCode instead of form.eventCode because when serving from the document viewer, form is undefined. Changing it back to form.eventCode will break one of the flows that Tenille pointed out in her feedback! We can chat about this when you have time!

Reply:
Thank you for catching this! Merge gone wrong, I meant to keep your changes!

Reply:
Awesome!